### PR TITLE
Ep integration

### LINKED ISSUE #36: IN: Profile Page > Edit Profile
Integrate the Profile page. On save and update, this should update the profile on the backend

### PR DESCRIPTION
**Pull request by hodge-jai**
### What this PR does (required):
- RESOLVES #36
- Integrates the FE profile edit form with the back end.

### Any information needed to test this feature (required):
- Sign up as a new user and edit profile.
- Test different inputs

### Any issues with the current functionality (optional):
- Will have to integrate #24 with current edit menu to add the AWS image url to the users profile record

### CODE REVIEW

**Review by sundayezeilo — COMMENTED** [review]
Good work on this UI. One thing you forgot in filling out the PR templates was to add a few screenshots of your result. Please always remember to do this as it's required. :smile:

File: client/src/components/DatePicker/DatePicker.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { useState, ChangeEvent } from 'react';
+import TextField from '@mui/material/TextField';
+import AdapterDateFns from '@mui/lab/AdapterDateFns';
+import LocalizationProvider from '@mui/lab/LocalizationProvider';
+import DesktopDatePicker from '@mui/lab/DesktopDatePicker';
+import { FieldProps } from 'formik';
+
+export default function MaterialUIPickers({ field, form, ...others }: FieldProps) {
+  const [value, setValue] = useState<Date | null>(new Date('2014-08-18T21:11:54'));
+  console.log(form);
```
Comment (rajivtitus) [inline client/src/components/DatePicker/DatePicker.tsx]: Please remove log to the console

File: client/src/components/EditProfile/ProfileForm/ProfileForm.tsx [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+      dateOfBirth: Date;
+      phoneNumber: string;
+      address: string;
+      description: string;
+    }>,
+  ) => {
+    editProfile({ firstName, lastName, gender, dateOfBirth, phoneNumber, address, description }).then((data) => {
+      if (data.error) {
+        setSubmitting(false);
+        updateSnackBarMessage('Something went wrong');
```
Comment (rajivtitus) [inline client/src/components/EditProfile/ProfileForm/ProfileForm.tsx:63]: NIce use of the snackbar!

**Comment by rajivtitus** [comment]
Good work! Just a minor comment from my end, I'll hold off on approving this just in case we're making any changes to the UI

**Review by rajivtitus — APPROVED** [review]
LGTM! I am approving this PR, please make sure you update the UI elements here as we've added labels within the edit profile form 👍🏽

**Pull request merged**